src/client/CartClient.ts

```diff
@@ -0,0 +1,95 @@
+import { APIConstants } from '../constants/APIConstants';
+import { CartData, DeleteCart } from '../types/Cart';
+
+import { BaseClient } from './BaseClient';
+
+export class CartClient extends BaseClient {
+  /**
+   * @ignore
+   */
+  public constructor(private readonly accessToken: string) {
+    super();
+  }
+
+  /**
+   * @param {string} cutomerId ClientCustomer Id
+   *
+   * @param {string} lineItemsAfter
+   * If `data.cart.pageInfo.hasNextPage` is `true`, then request next page by passing
+   * `data.cart.pageInfo.endCursor` as `lineItemsAfter` parameter. You can also pass `data.cart.edges[i].cursor`
+   * as `lineItemsAfter` parameter to get results after that cursor.
+   *
+   * @returns {Promise<CartData>}
+   * Paginated cart list of maximum 250 cart items. To request further cart items, use `lineItemsAfter` parameter.
+   */
+  async getByClientCustomerId(cutomerId: string, lineItemsAfter?: string): Promise<CartData> {
+    const params = new Object({
+      customerId: cutomerId,
+      lineItemsAfter: lineItemsAfter,
+    });
+
+    const url = `${APIConstants.API_BASE_URL}/carts`;
+
+    return this._get<CartData>(url, params, {
+      [APIConstants.ZECKO_ACCESS_TOKEN_HEADER_KEY]: this.accessToken,
+    });
+  }
+
+  /**
+   * @param {JSON} cartActionRequest CartActionRequest
+   * @property `customerId`
+   * ClientCustomer Id - string property of cartActionRequest
+   * @property `variantId`
+   * Product Variant Id - string property of cartActionRequest
```

Comment:
`string property of cartActionRequest` should not be needed if you use interface

Reply:
Okayy